tgbotapi.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/send/RepliesWithChatsAndMessages.kt

```diff
@@ -2045,3 +2048,43 @@ suspend fun TelegramBot.reply(
         )
     }
 }
+
+suspend fun TelegramBot.reply(
+    toChatId: IdChatIdentifier,
+    toMessageId: MessageId,
+    content: PaidMediaInfoContent,
```

Comment:
Here must be next args:

```kotlin
    toChatId: IdChatIdentifier,
    toMessageId: MessageId,
    starCount: Int,
    media: List<TelegramPaidMedia>,
    entities: TextSourcesList,
    showCaptionAboveMedia: Boolean = false,
    threadId: MessageThreadId? = chat.id.threadId,
    businessConnectionId: BusinessConnectionId? = chat.id.businessConnectionId,
    disableNotification: Boolean = false,
    protectContent: Boolean = false,
    allowSendingWithoutReply: Boolean? = null,
    replyMarkup: KeyboardMarkup? = null
```

Besides, it is required to add overload with `text + parseMode` instead of `entities`